pkgs/data/fonts/ohxproto/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "0xproto";
+  version = "1.300";
+  underscoreVersion = builtins.replaceStrings [ "." ] [ "_" ] version;
+
+  src = fetchzip {
```

Let's put this inside a let block of the source, just to make sure no one confuses it for an actual argument of `mkDerivation`.

```suggestion
  src = let
    underscoreVersion = builtins.replaceStrings [ "." ] [ "_" ] version;
  in fetchzip {
```